src/applications/vaos/covid-19-vaccine/flow.js

```diff
@@ -81,3 +82,85 @@ export default {
     next: '',
   },
 };
+
+/**
+ * Function to get new COVID appointment workflow.
+ * The URL displayed in the browser address bar is changed when the feature flag
+ * is true.
+ *
+ * @export
+ * @param {boolean} state - New COVID appointment state
+ * @returns {object} COVID appointment workflow object
+ */
+export default function getNewBookingFlow(state) {
+  const featureBreadcrumbUrlUpdate = selectFeatureBreadcrumbUrlUpdate(state);
+
+  return {
+    ...flow,
+    clinicChoice: {
+      ...flow.clinicChoice,
+      url: featureBreadcrumbUrlUpdate
+        ? 'clinic'
+        : '/new-covid-19-vaccine-appointment/choose-clinic',
+    },
+    contactFacilities: {
+      ...flow.contactFacilities,
+      url: featureBreadcrumbUrlUpdate
+        ? 'contact-facility'
+        : '/new-covid-19-vaccine-appointment/contact-facility',
+    },
+    contactInfo: {
+      ...flow.contactInfo,
+      url: featureBreadcrumbUrlUpdate
+        ? 'contact-information'
+        : '/new-covid-19-vaccine-appointment/contact-info',
+    },
+    home: {
```

Comment:
Using the attribute `home` and `root` have a similar connotation. How about changing `home` to `typeOfCare` or  `chooseCare` or `scheduleCare`

Reply:
@jenniemc - Great suggestion!